frontend/src/components/DeleteFund.js

```diff
@@ -0,0 +1,29 @@
+const DeleteFund =({currUser, id, setFunds, funds })=>{
+    const logout=async (id)=>{
+        try {
+            const response=await fetch(`http://localhost:3000/funds/${id}`,{
+                method: "delete",
+                headers: {
+                    "content-type": "application/json",
+                    "authorization": localStorage.getItem("token")
+                },
+            })
+            const data=await response.json()
+            if(!response.ok) throw data.error
+            console.log(funds)
+            setFunds(funds.filter(item => item.id !== id))
+        } catch (error) {
+            console.log("error", error)
+        }
+    }
```

Comment:
you have a very common behavior that happens in all these files.  It often is good to try to extract things when you see patterns arise.  It allows you to reuse code so that next time it is easier to use, and it helps you to track down issues and make changes faster since it is all in one place.